docs/Support/faq.md

```diff
@@ -66,31 +146,109 @@ Yes! See [boolean logic](/accessControl/conditionTypes/booleanLogic) for example
+```
+
+### 11. What’s the maximum number of accessControlConditions allowed at once?
+
+30
+
+### 12. Where can I save the encryptedString & encryptedSymmetricKey?
+
+The Lit network doesn’t store these encrypted contents for you. You can store these anywhere you want; in a database, in an on-chain smart contract, IPFS, anywhere else you like.
+
+### 13. How to construct an accessControlCondition to authorize only a specific wallet address?
+
+Check out the solution our docs [here](https://developer.litprotocol.com/coreConcepts/accessControl/EVM/basicExamples#a-specific-wallet-address).
+
+### 14. How to use a time-lock based accessControlCondition?
+
+Check out the solution our docs [here](https://developer.litprotocol.com/coreConcepts/accessControl/EVM/timelock).
+
+<br />
+
+## Design Patterns
+
+### 1. How to allow only permitted users to execute a LitAction?
+
+Use the `PKPPermission.addPermittedAddress()` to allow only specific users to execute a LitAction. Note, this will allow these users to execute any LitAction.
+
+For other permissions, please see the contract [here](https://github.com/LIT-Protocol/LitNodeContracts/blob/main/contracts/PKPPermissions.sol).
+
+### 2. How to allow permitted users to execute only specific LitActions?
+
+Assign the PKP to itself as we don’t want the PKP owner to arbitrarily change the functioning of the LitAction. Use the PKPHelper.`mintNextAndAddAuthMethods()` for that & to initially setup allow only specific IPFS CIDs to execute.
+
+But anyone can now call these LitActions, so how should one add a permitted list of users? We can store the permitted list of users, either on-chain or the LitAction, can fetch it from there or put that in the LitAction itself and use `conditionalSigning` to check whether the provided AuthSig is permitted to execute the LitAction. Learn more [here](https://developer.litprotocol.com/coreconcepts/litactionsandpkps/actions/conditionalsigning/).
+
+### 3. But I want to upgrade the permitted LitActions/users?
+
+Since the PKP is assigned to itself in the setup stage as described above, we can’t directly permit any new users/LitActions. Thus we have to lay down the upgrade logic in the LitAction itself. We can have an admin user that can satisfy the update AuthSig check & upgrade the LitAction to a new IPFS CID which will have the new permitted addresses/code.
+
+### 4. I want to create multiple different AuthSigs but don't want my users to sign multiple times?
+
+Firstly, if you are using Lit in a browser, the AuthSigs is stored in its local storage so you don’t need to sign multiple times as the stored AuthSig will be used subsequently. This design pattern is specifically applicable to when you want to sign different SIWE resources or messages which is generally required in [Custom Contract Calls](https://developer.litprotocol.com/coreConcepts/accessControl/EVM/customContractCalls#using-siwe-params-in-custom-contract-calls).
```

Comment:
"the AuthSig" (singular in the first sentence

Reply:
ty @debbly - updated!